app/routes/frontpage/components/PublicFrontpage.css

```diff
@@ -5,21 +5,26 @@
   grid-gap: 40px;
   grid-template-columns: 1fr 1fr 1fr;
   grid-auto-rows: auto;
-  grid-template-areas: 'welcome welcome login' 'events events hsp' 'article article readme' 'links links links';
+  grid-template-areas: 'banner banner banner' 'welcome welcome login' 'events events hsp' 'article article readme' 'links links links';
 
   @media (--mobile-device) {
     grid-gap: 20px;
     grid-template-columns: 1fr;
-    grid-template-areas: 'welcome' 'login' 'events' 'hsp' 'article' 'readme' 'links';
+    grid-template-areas: 'banner' 'welcome' 'login' 'events' 'hsp' 'article' 'readme' 'links';
```

Comment:
Nice catch, I didn't notice that. It doesn't seem like there is any way to avoid the gap when using `grid-template-areas`, I guess we could just use a flexbox instead maybe for the banner